examples/test_examples.py

```diff
@@ -64,6 +65,15 @@ def get_setup_file():
     return args.f
 
 
+def get_results(output_dir):
+    results = {}
+    path = os.path.join(output_dir, "all_results.json")
+    if os.path.exists(path):
+        with open(path, "r") as f:
+            results = json.load(f)
+    return results
```

Comment:
if we are expecting this to always work, then perhaps:
```
else:
    raise ValueError(f"can't find {path}")
```

otherwise `result["eval_accuracy"]` will complain about a missing key, which is not the problem.